lib/ansible/modules/cloud/ovirt/ovirt_disk.py

```diff
@@ -430,6 +450,7 @@ def build_entity(self):
                     name=self._module.params.get('storage_domain'),
                 ),
             ],
+            quota=otypes.Quota(id=self._module.params.get('quota')),
```

Comment:
quota=otypes.Quota(
    id=self..param('quota')
) if self.param('quota') else None

Not sure, but I think you should keep it None in case user didn't pass the parameter.

Reply:
It works in both cases, with and without the assignment to None in case the parameter 'quota' is not defined, if not defined the disk image is assigned to the 'default' quota.
But if you want I could push a commit with suggested changes.

Reply:
If you verified it's ok, thanks.

Reply:
I think it would be better, because with current code when user don't specify quota, the Python SDK will send a request similar to this:

```xml
<disk>
  <alias>mydisk1</alias>
  <size>123</size>
  <storage_domains>
      <storage_domain>
       <name>mysd</name>
      </storage_domain>
  </storage_domains>
  <quota/>
<disk>
```

So it will send empty `quota` element, which is useless, but fortunatelly doesn't cause any issue at oVirt backend.